shopping_mall/backend/app/routers/chatbot.py

```
@@ -64,9 +180,264 @@ def rate_chat_log(log_id: int, body: ChatRating, db: Session = Depends(get_db)):
+            .filter(ChatSession.user_id == body.user_id, ChatSession.status == "active")
+            .first()
+        )
+        if existing_session:
+            session = existing_session
+        else:
+            # Fallback: shouldn't happen, but retry once more
+            raise HTTPException(
+                status_code=500,
+                detail="Failed to create or retrieve active chat session",
+            )
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)